happy_bank_core/data/file_connector.py

```diff
@@ -0,0 +1,37 @@
+from happy_bank_core.data.connector import Connector
+from happy_bank_core.logic.account import Account
+import json
+import logging
+import os
+
+logger = logging.getLogger(__name__)
+
+abs_path = os.path.abspath("data/customers.json")
```

Comment:
Do we really need this as a global variable? Could we encapsulate it and wrap it somewhere it belongs?